src/components/Comment.jsx

```diff
@@ -1,11 +1,43 @@
 import { Parser, HtmlRenderer } from "commonmark";
+import jsCookies from "js-cookies";
+import { useContext, useEffect, useState } from "react";
+import authorService from "../services/author";
+import postService from "../services/post";
+import { UserContext } from "../UserContext";
 import MiniProfile from "./MiniProfile";
 
 const Comment = ({ comment }) => {
+  const { user } = useContext(UserContext);
+  const [ commentState, setCommentState ] = useState(comment);
+
   const CMParser = new Parser({ safe: true });
   const CMWriter = new HtmlRenderer();
 
-  console.log(comment)
+  useEffect(() => {
+    console.log(comment)
+    if (comment?.id === undefined) return;
+    const getCommentLikes = async () => {
+      const res = await postService.getCommentLikes(comment.id.split("/").at(-5), comment.id.split("/").at(-3), comment.id.split("/").at(-1));
+      console.log(res);
```

Comment:
remove console.log()

Reply:
well caught team mate. i will get to this at once.